src/zcl_logger_factory.clas.abap

```diff
@@ -48,6 +48,16 @@ CLASS zcl_logger_factory DEFINITION
       RETURNING
         VALUE(r_display_profile) TYPE REF TO zif_logger_display_profile.
 
+
+    "! Reopens specific log instance.
+    CLASS-METHODS open_log_by_db_number
+      IMPORTING
+        db_number        TYPE balognr
+        settings         TYPE REF TO zif_logger_settings OPTIONAL
+      RETURNING
+        VALUE(logger_if) TYPE REF TO zif_logger.
```

Comment:
Could you use r_log in order to be consistent with create_log and open_log?

Reply:
Sure